src/lib/circle.js

```diff
@@ -0,0 +1,179 @@
+const clipboard = require('clipboardy');
+const fs = require('fs');
+const inquirer = require('inquirer');
+const path = require('path');
+const request = require('request-promise');
+const validator = require('validator');
+const yaml = require('js-yaml');
+
+const passwordStorage = require('./password-storage')('circleci');
+
+const circleConfig = {
+  version: 2,
+  jobs: {
+    build: {
+      docker: [
+        {
+          image: 'circleci/node:latest',
+        },
+      ],
+      steps: [
+        'checkout',
+        {
+          run: {
+            name: 'install',
+            command: 'npm install'
+          }
+        },
+        {
+          run: {
+            name: 'release',
+            command: 'npm run semantic-release || true',
+          },
+        },
+      ],
+    },
+  },
+};
+
+async function getUserInput(info) {
+  return await inquirer.prompt([
+    {
+      type: 'input',
+      name: 'token',
+      message: 'What is your CircleCI API token?',
+      validate: input => input.length === 40 ? true : 'Invalid token length',
+      default: async answers => {
+        const clipboardValue = await clipboard.read();
```

Comment:
Does it allow the user to confirm they the actually want to use the content of the clipboard? Does it allow them to enter something else?
If yes, that's a really clever feature 👍

Reply:
Yes, it's just shown as the default, exactly like npm or github username work:
`? What is your CircleCI API token? (abvdfi89.......)`
And the user can type a new value, or press enter to accept the default.

Reply:
Really nice!